.release-it.json

```diff
@@ -1,9 +1,9 @@
 {
     "hooks": {
-        "after:bump": "./.github/helpers/update-saucectl-template.sh"
+        "after:bump": "bash ./scripts/update-playwright-browsers.sh"
     },
     "git": {
-        "commit": false,
+        "commit": true,
```

Comment:
Any sideeffects from this one? 🤔

Reply:
I don't think so. `release-it` requires a clean working dir before starting. That should ensure only files modified by the release-it process will be committed.

But now that you mention it, I'm wondering if changes can be pushed by a github action...need to investigate.

Reply:
Pushing from a github action should work.

Reply:
@alexplischke there is one side effect I've found.

Part of the `release-it` workflow is to bump the version in `package.json`. Because we have `git.commit: false` configured, that version bump never gets committed and our `package.json` has been stuck at `0.0.0`. If that is intentional, that behaviour will change. 

Pinging @FriggaHel since he did the initial implementation of the release workflow.

Reply:
The initial pipeline was working without any commit. When moving to `release-it`, we kept the same logic. 
That's why we kept version 0.0.0 in the `packages.json`.

To be consistent with other images, I would be in favor of keeping the same behaviour everywhere, and avoid adding a bumping commit.